Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -301,40 +346,6 @@ public void StopClimbing()
 		}
 	}
 
-	protected void CheckGroundSlope()
```

Comment:
The lifter/bumper combo should take care of this. If the slope is steep enough Then it will collide with the bumper and push the player back from the wall. We can tweak the maximum climbable slope by changing the configuration of the colliders.